packages/sdk-core/src/model/http/SubmissionUrlInformation.ts

```diff
@@ -71,4 +87,15 @@ export class SubmissionUrlInformation {
 
         return url.searchParams.get('token');
     }
+
+    public static changeSubmissionFormat(submissionUrl: string, desiredFormat: 'json' | 'plcrash' | 'minidump') {
+        const submitIndex = submissionUrl.indexOf(this.SUBMIT_PREFIX);
+        const url = new URL(submissionUrl);
+        if (submitIndex !== -1) {
+            url.pathname = url.pathname.replace('/json', `/${desiredFormat}`);
```

Comment:
How would you know if the submission URL had `/json` to begin with?

Reply:
if it doesn't then we're dealing with the invalid URL and we cannot do anything with it

Reply:
but this is not true. If you replace `/json` with `/minidump`, is this now an invalid URL? Because if I pass it, and ask the function to replace this back to `json`, it will not work.

Reply:
Modified the algorithm to use the last part of the URL instead

Reply:
What if someone passes an URL with trailing slash?
https://submit.backtrace.io/name/000000000000a1eb7ae344f6e002de2e20c81fbdedf6991c2f3bb45b11111111/json/

I guess the implementation will break then.

Reply:
For this specific use case I've changed it to pathname split